app/src/main/java/com/perflyst/twire/service/Settings.java

```diff
@@ -59,6 +59,7 @@
     private final String CHAT_LANDSCAPE_ENABLE = "chatLandscapeEnable";
     private final String CHAT_LANDSCAPE_SWIPEABLE = "chatLandscapeSwipable";
     private final String CHAT_LANDSCAPE_WIDTH = "chatLandscapeWidth";
+    private final String CHAT_ENABLE_SSL = "chatenablessl";
```

Comment:
Can you make the capitalization consistent?
```suggestion
    private final String CHAT_ENABLE_SSL = "chatEnableSSL";
```

Reply:
Fixed in: https://github.com/twireapp/Twire/pull/226/commits/66a79c8ef2c86c3ff78a35e108785bc89679f95a